subprojects/core/src/integTest/groovy/org/gradle/api/NestedConfigureDslIntegrationTest.groovy

```diff
@@ -231,7 +229,7 @@ assert repositories.size() == 1
         errorOutput.contains("Could not find method maven() for arguments")
     }
 
-    @ToBeFixedForConfigurationCache(skip = INVESTIGATE)
+    @ToBeFixedForConfigurationCache(because = "resolves configuration at execution time through configuration container")
```

Comment:
❔  Can the test still be fixed for CC or is resolving the configuration at execution time inherent to the test?

Reply:
> ❔ Can the test still be fixed for CC or is resolving the configuration at execution time inherent to the test?

I've tried, but it breaks output checking. The root cause is the same (Closure delegate not being available), just the message is slightly different.